ee/query-service/usage/manager.go

```diff
@@ -139,6 +139,8 @@ func (lm *Manager) UsageExporter(ctx context.Context) {
 			lm.CollectAndStoreUsage(ctx)
 		case <-uploadTicker.C:
 			lm.UploadUsage(ctx)
+			// remove the old snapshots
+			lm.repository.DropOldSnapshots(ctx)
```

Comment:
@nityanandagohain shouldn't `DropOldSnapshots` trigger when `UploadUsage` does not return an error? I see returning error is already implemented in `UploadUsage`

Reply:
ok